### ISSUE TITLE
write README in plain English

### ISSUE THREAD

**Issue by gulping** [redox-os/kernel#55]
README of this project is highly suggestive to all programmers not only contributors!

　

However, I (who unfamiliar to English and this project) cannot understand some sentences.

> it gives severe backslash later on.

I cannot understand what "backslash" means.

> leading to UB

I cannot understand what does "UB" stands for.

　

So I hope that write README in easier English.

　

thanks.

**Comment by jackpot51** [comment]
Thanks, I will update the wording.

**Comment by jackpot51** [comment]
Closing in favor of #53
